external-crates/move/crates/move-compiler/tests/linter/incorrect_constant_naming.move

```diff
@@ -0,0 +1,16 @@
+module 0x42::M {
+    // Incorrectly named constants
+    const Another_BadName: u64 = 42; // Should trigger a warning
+    const YetAnotherName: u64 = 777; // Should not trigger a warning
+
+    const minimumValue: u64 = 10; 
+    const some_important_number: u64 = 55;
```

Comment:
I should have caught this a lot earlier, sorry. 
These constants have invalid names. There is no need to lint against constants where the name is an error.

Reply:
refactored